packages/mermaid/package.json

```diff
@@ -87,7 +87,9 @@
     "roughjs": "^4.6.6",
     "stylis": "^4.3.6",
     "ts-dedent": "^2.2.0",
-    "uuid": "^11.1.0"
+    "uuid": "^11.1.0",
+    "@mermaid-js/layout-elk": "workspace:^",
+    "@mermaid-js/layout-tidy-tree": "workspace:^"
```

Comment:
bundling this with mermaid defeats the purpose of splitting it into different packages. 

The implementors will have to add them separately during mermaid init.